File: 진우/8weeks/10101. 삼각형 외우기/삼각형 외우기.py
arr = [int(input()) for _ in range(3)]

total = sum(arr)

if total != 180:
    print("Error")
elif arr[0] == arr[1] == arr[2]:
    print("Equilateral")
elif arr[0] == arr[1] or arr[1] == arr[2] or arr[0] == arr[2]:
    print("Isosceles")
else:
    print("Scalene")

File: 진우/8weeks/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
#include <vector>
#include <cmath>
#include <algorithm>

using namespace std;

int main() {
	int arr[3];

	for (int i = 0; i < 3; i++) {
		cin >> arr[i];
	}

	int sum = 0;
	for (int i = 0; i < 3; i++) {
		sum += arr[i];
	}

	if (sum != 180) {
		cout << "Error" << "\n";
	}
	else if (arr[0] == arr[1] && arr[1] == arr[2]) {
		cout << "Equilateral" << "\n";
	}
	else if (arr[0] == arr[1] || arr[1] == arr[2] || arr[0] == arr[2]) {
		cout << "Isosceles" << "\n";
	}
	else {
		cout << "Scalene" << "\n";
	}

	return 0;
}
